README.md

```diff
@@ -1,5 +1,48 @@
+# Data Catalog React
+
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+This is a REACT frontend designed to utilize the latest stable version of [DKAN 2.x](https://github.com/GetDKAN/dkan) as a backend.
+
+This application serves as a starter app, or example of how to use the [data-catalog-components](https://github.com/GetDKAN/data-catalog-components) library to easily create open data catalogs.
+
+## Auto Set Up
+1) Follow the [DKAN Tools](https://github.com/GetDKAN/dkan-tools) README to stand up the backend. Include the `--demo` flag to have the frontend installed and example pages built as well.
+
+## Manual Set Up
+If you have a backend already running and just need the frontend:
+
+1) Clone this repository in your **docroot** ``git clone https://github.com/GetDKAN/data-catalog-react.git``
+1) Install the dependencies with [npm](https://www.npmjs.com/):
+   1) ``cd data-catalog-frontend``
```

Comment:
- Should we mention to clone into `docroot/frontend` to be able to use dkan tools commands?
- Replace 'data-catalog-frontend' with 'data-catalog-react'
- Add info about .env file(s)